test/cppsuite/test_harness/workload/workload_validation.h

```diff
@@ -38,7 +38,11 @@ extern "C" {
 #include "database_model.h"
 
 namespace test_harness {
-
+struct key_state {
+    bool exists;
+    std::string value;
```

Comment:
You probably want key_value_t instead of string

Reply:
8153016